### DISCUSSION TITLE
Avalonia 11.0 support

### DISCUSSION THREAD

**Discussion by jp2masa** [AvaloniaUI/AvaloniaEdit#242] [General]
I was trying to get AvaloniaEdit running on latest Avalonia nightly builds, and noticed there wasn't any support, so I made some updates to get it working, it's in [this branch](https://github.com/jp2masa/AvaloniaEdit/tree/avalonia-11.0).

In the meanwhile I noticed there's a `feature/TextFormatterPort` branch, so probably my work won't be of much help anyway, just wanted to let you know.

Also, didn't check if it's fixed in the `feature/TextFormatterPort` branch, but there's a bug with hovering text which shows normal cursor instead of text one, but this seems to be a bug of the new Avalonia compositor, because using `.With(new Win32PlatformOptions() { UseCompositor = false })` works fine.

**Comment by Gillibald** [comment]
Avalonia 0.10.999 support is available via https://github.com/AvaloniaUI/AvaloniaEdit/tree/dev

**Reply by jp2masa** [reply]
No idea how I missed that, thanks!

**Comment by KrzysztofDusko** [comment]
Is support for Avalonia 11.0.0 Preview1 planed ?

**Reply by danipen** [reply]
Already supported on master.

**Comment by Numpsy** [comment]
Would it be possible to get a new preview release of AvaloniaEdit with the Avalona 11 preview 6 changes? (i looked in the avalonia nightly feed, and didn't anything up to date in there)
